src/visitors/kinetic_block_visitor.hpp

```diff
@@ -14,8 +14,8 @@
 
 #include <map>
 #include <memory>
-#include <set>
 #include <string>
+#include <unordered_set>
```

Comment:
why the switch?

Reply:
There was no reason to have a set. Unordered sets perform better for finding nodes (required in the routines even before my addition)